requestvideoframecallback/main.js

```diff
@@ -0,0 +1,46 @@
+const startDrawing = () => {
+  const button = document.querySelector("button");
+  const video = document.querySelector("video");
+  const canvas = document.querySelector("canvas");
+  const ctx = canvas.getContext("2d");
+  const fpsInfo = document.querySelector("#fps-info");
+  const metadataInfo = document.querySelector("#metadata-info");
+
+  button.addEventListener("click", () =>
+    video.paused ? video.play() : video.pause()
+  );
+
+  video.addEventListener("play", () => {
+    if (!("requestVideoFrameCallback" in HTMLVideoElement.prototype)) {
+      return alert(
```

Comment:
Do you want to do this:
```suggestion
      console.error(
```